### ISSUE TITLE
Program Step - Problem Page

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#164]
:warning:  Wire-frames and live-demos are just for example purpose.  **They doesn't meet entirely this requirement**.  

:pencil2: Pencil icon marks specs that are waiting for definition

****************
# Index
1. Content Area
    1. Layout variantes
    1. Drag-and-Drop area
1. Preview Mode
    1. Layout variantes

****************
This specification is about the template "Content Page" and complements #143 specification.

# Content Area
![image](https://user-images.githubusercontent.com/993369/83819402-1aa1a200-a6a0-11ea-9ea7-270eccd87c82.png)

### Layout Variants
Layout's variants conditional to quantity of items or CoursePlan[Template]
### Drag-and-drop area
Each box of these layouts are drag-and-drop areas that only accepts specific type of components
| area name | components restrictions |
| -- | -- | 

# Preview Mode

![image](https://user-images.githubusercontent.com/993369/83819411-242b0a00-a6a0-11ea-8028-79c2058c6191.png)

### Layout Variants
Layout's variants conditional to quantity of items or CoursePlan[Template]

**Comment by juantenesaca** [comment]
@ecureuill What is the situation of the transcription of the audio. I think that @CAndresH told me that it will not go.

**Comment by ecureuill** [comment]
> @ecureuill What is the situation of the transcription of the audio. I think that @CAndresH told me that it will not go.

@juantenesaca I'm not sure if I understood what you mean.


When you said that @CAndresH told you that "it eill not go" you mean that the feature is not going to be coded? 

From my side, the specification is ready to be coded.

**Comment by ecureuill** [comment]
@juantenesaca it's expected to have the transcription. I'm not aware of why it was removed.

**Comment by juantenesaca** [comment]
@ecureuill Andres (@CAndresH ) told me you said to him that it has to be removed from accessibility window on the tutor side. So if the tutor is not allowed to add a transcription, the student in consequence will not have it.

**Comment by CAndresH** [comment]
@ecureuill  @juantenesaca ,Yes, Camila, do you remember that in the audio part you told me that only text alternatives should go, not audio transcription? on the teacher's side

**Comment by ecureuill** [comment]
Hi @CAndresH  I remember that!
I found the whatsapp message I sent you.

The problem was this tab "audio transcription", that should not exist.
![image](https://user-images.githubusercontent.com/993369/93387747-1ad54000-f840-11ea-9506-ef2cd9acbf31.png)

But the "transcription" concept is covered by the "text alternative" long field

![image](https://user-images.githubusercontent.com/993369/93387830-33455a80-f840-11ea-91cb-9004ac7f1ba3.png)

**Comment by juantenesaca** [comment]
@ecureuill So we have just to rename "Audio transcription" instead of "Long description".

**Comment by ecureuill** [comment]
In teacher module: remove the "audio transcription" tab, because the transcription information goes in "text alternative" tab

In student module: render as in this specification

**Comment by juantenesaca** [comment]
@ecureuill It is not completed, but please review then I could fix the specs missing.
